storage/google/cloud/storage/acl.py

```diff
@@ -196,7 +196,7 @@ class ACL(object):
         'bucketOwnerFullControl',
     ])
     """See:
-    https://cloud.google.com/storage/docs/access-control#predefined-acl
+    https://cloud.google.com/storage/docs/access-control/
```

Comment:
1. This would not be a 301?
1. It seems like this content has moved: https://cloud.google.com/storage/docs/access-control/lists#predefined-acl